roachtest: use noprepare for multi-region TPC-C

otan opened the pull request:
Simple mode errors right now; noprepare restore parity with past 
state.

Resolves #69471
Resolves #69472

Release justification: test only change

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/69676)
<!-- Reviewable:end -->

rafiss reviewed (approved):
fine to merge it, but could you share a bit about the evidence you've seen so far?

i think it would help me try to debug the weird stuff in https://github.com/cockroachdb/cockroach/issues/69411#issuecomment-909816996 as well

otan commented:
> could you share a bit about the evidence you've seen so far?

lol well, i actually just dug into this a bit deeper and found: `E210827 12:56:09.611657 1 workload/cli/run.go:521  [-] 850  error in newOrder: Cannot encode []interface {} in simple protocol - []interface {} must implement driver.Valuer, pgtype.TextEncoder, or be a native type`

otan commented:
so i guess i should just learn to read!

in the meantime, let the chaos continue

bors r=rafiss

otan committed:
roachtest: use noprepare for multi-region TPC-C

Simple mode errors right now; noprepare restore parity with past
state.

Release justification: test only change

Release note: None

otan commented:
oops, forgot to push!

bors r=rafiss

rafiss commented:
ah i see

where is the `[]interface{}` type being used in TPCC? i can't find it. it should be possible to fix by using `pgtype.*Array` instead.
